tilequeue/command.py

```diff
@@ -1074,6 +1074,19 @@ def tilequeue_prune_tiles_of_interest(cfg, peripherals):
                     coord_marshall_int(deserialize_coord(l.strip()))
                     for l in f
                 )
+        elif 's3' in info:
+            from boto import connect_s3
+            from boto.s3.bucket import Bucket
+            s3_conn = connect_s3()
+            bucket = Bucket(s3_conn, info['s3']['bucket'])
+            key = bucket.get_key(info['s3']['key'])
+            raw_coord_data = key.get_contents_as_string()
+            for line in raw_coord_data.splitlines():
+                coord = deserialize_coord(line.strip())
+                if coord:
+                    coord_int = coord_marshall_int(coord)
+                    uplifted_coord_int = coord_int_zoom_up(coord_int)
```

Comment:
@iandees the coordinates that are currently in the file are what get requested, so they need to get uplifted. Do you think it makes more sense to have that happen on the way into the file in the first place? Or do you think it's better to handle it here when reading them as it stands now? It's weird that it's the only path that reads tiles and does the uplift.